.github/workflows/node.js.yml

```diff
@@ -26,4 +26,4 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
       - run: npm ci --legacy-peer-deps
-      - run: npm test
+      - run: npm run test:lib
```

Comment:
Now that that there is `test:lib` and `test:playwright`, do we not want both to run in the CI?

Reply:
'test:lib' runs for multiple Node versions. For 'test:playwright' we can know which Node version is running in CI and the tested output ist static html so I think the current setup makes sense.

We should think about testing other browsers in the future but probably get one running first (the screenshot test will get harder to match with Firefox).

Reply:
Ah I had missed that playwright.yml runs as a separate action on CI. I had just given a quick look at this diff